Vagrantfile

```diff
@@ -79,9 +79,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       inline: "pip install -U django netaddr ipython"
     server.vm.provision "shell",
       run: "always",
+      inline: "echo '1' > /proc/sys/net/ipv4/ip_forward"
+    server.vm.provision "shell",
+      run: "always",
       inline: "cd /vagrant/vpn-proxy/ && " \
+              "./manage.py makemigrations && " \
```

Comment:
`makemigrations` is something you run locally when you are developing. This will generate some python files inside the `app/migrations` directory. These files should then be commited to the repo. Anyone checking out/pulling the branch should only need to run `migrate`.